docs/en/api/FS-API.md

```diff
@@ -296,8 +288,8 @@ this policy preference exists only for local workers, not remote workers; remote
 blocks to the highest tier.
 
 By default, data is written to the top tier. Users can modify the default setting through the
-`alluxio.user.file.write.tier.default` [configuration]({{ '/en/operation/Configuration.html' | relativize_url }})
-property or override it through an option to the 
+`alluxio.user.file.write.tier.default` [property]({{ '/en/reference/Properties-List.html' | relativize_url }})
```

Comment:
The original link doesn't contain the mentioned property here. It's quite confusing to me why we are linking to the parent page of all configurations here. So I changed it to link to the page that contains this specific property.